android-mvvm-kotlin/app/src/main/res/layout/activity_main_sample.xml

```diff
@@ -14,14 +15,16 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:background="@color/white"
-        android:orientation="vertical"
-        tools:context=".samples.ui.container.MainSampleActivity">
```

Comment:
define context to click come back from layout to kotlin code file

Reply:
yes